devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy

```diff
@@ -300,6 +355,47 @@ private command(physicalgraph.zwave.Command cmd) {
 	}
 }
 
+def getConfigurationCommands() {
+	log.debug "getConfigurationCommands"
+	def result = []
+	if (isEverspringSP817()) {
+		if (!state.retriggerIntervalSettings) state.retriggerIntervalSettings = 180 // default value (parameter 4) for Everspring SP817
+		Integer retriggerIntervalSettings = (settings.retriggerIntervalSettings as Integer) ?: 180 // default value (parameter 4) for Everspring SP817
+		if (!isConfigured() || (retriggerIntervalSettings != state.retriggerIntervalSettings)) {
+			state.configured = false
+			if (!state.intervalConfigured || retriggerIntervalSettings != state.retriggerIntervalSettings) {
+				state.intervalConfigured = false
+				result << zwave.configurationV2.configurationSet(parameterNumber: 4, size: 2, scaledConfigurationValue: retriggerIntervalSettings)
+				result << zwave.configurationV2.configurationGet(parameterNumber: 4)
+			}
+		}
+	}
+	return result
+}
+def lateConfigure() {
+	log.debug "lateConfigure"
+	sendHubCommand(getConfigurationCommands(),200)
+}
+private isConfigured() {
+	return state.configured == true
+}
+def zwaveEvent(physicalgraph.zwave.commands.configurationv2.ConfigurationReport cmd) {
+	if (isEverspringSP817()) {
+		if (cmd.parameterNumber == 4) {
+			state.retriggerIntervalSettings = scaledConfigurationValue
+			state.intervalConfigured = true
```

Comment:
since there's only one parameter you probably don't need two booleans to track whether the device is configured

Reply:
Sure, fixed